conf/queues.yaml

```diff
@@ -1,30 +1,40 @@
 partitions:
-  - name: default
+  -
+    name: default
+    placementrules:
+      - name: tag
+        value: namespace
+        create: true
     queues:
       - name: root
-        submitacl: "*"
-        queues:
-          - name: advertisement
-            resources:
-              guaranteed:
-                memory: 500000
-                vcore: 50000
-              max:
-                memory: 800000
-                vcore: 80000
-          - name: search
-            resources:
-              guaranteed:
-                memory: 400000
-                vcore: 40000
-              max:
-                memory: 600000
-                vcore: 60000
-          - name: sandbox
-            resources:
-              guaranteed:
-                memory: 100000
-                vcore: 10000
-              max:
-                memory: 100000
-                vcore: 10000
\ No newline at end of file
+        submitacl: '*'
+#partitions:
+#  - name: default
+#    queues:
+#      - name: root
+#        submitacl: "*"
+#        queues:
+#          - name: advertisement
+#            resources:
+#              guaranteed:
+#                memory: 500000
+#                vcore: 50000
+#              max:
+#                memory: 800000
+#                vcore: 80000
+#          - name: search
+#            resources:
+#              guaranteed:
+#                memory: 400000
+#                vcore: 40000
+#              max:
+#                memory: 600000
+#                vcore: 60000
+#          - name: sandbox
+#            resources:
+#              guaranteed:
+#                memory: 100000
+#                vcore: 10000
+#              max:
+#                memory: 100000
+#                vcore: 10000
```

Comment:
Can we remove the commented out config and make sure that we have a newline at the end of the file.

Reply:
Fixed.